redis/redis_migrator.rb

```diff
@@ -0,0 +1,47 @@
+# Original Author: Matt Casper (@mcasper/mattc@procore.com)
+
+require 'redis'
+
+# These can be new Redis servers, these can be different namespaces on the same
+# Redis server, whatever. Replace these with the appropriate Redis connections.
+# All keys from `old_redis` will be moved with the appropriate values to
+# `new_redis`.
+old_redis = Redis.new
```

Comment:
But if they need to connect using a namespace, they're going to have to pull in another gem (which should maybe be included here by default) and specifically use Ruby classes to get the namespace.